alembic.ini

```diff
@@ -29,7 +29,7 @@ script_location = db_migrate
 # are written from script.py.mako
 # output_encoding = utf-8
 
-sqlalchemy.url = postgres://codechecker@localhost:8764/codechecker
+sqlalchemy.url = postgres://codechecker@localhost:5432/codechecker
```

Comment:
Is this the default port? Should this work, when the used port is not the defult?